test/tests/integration/backend.js

```diff
@@ -0,0 +1,84 @@
+import chai from "chai";
+const request = require("supertest");
+
+const { app, updateLumensCache } = require("../../../backend/app.js");
+describe("integration", function() {
+  // update caches
+  before(async function() {
+    await updateLumensCache();
+  });
+
+  describe("backend api endpoints", function() {
+    it("/api/lumens should return successfuly with data", async function() {
+      let { body } = await request(app)
+        .get("/api/lumens")
+        .expect(200);
+
+      chai.expect(body).to.be.an("object");
+      chai.expect(Object.keys(body).length).to.not.equal(0);
```

Comment:
Is there a consistent set of keys we expect in this object?

Reply:
yes, I'm checking them in the unit test
https://github.com/stellar/dashboard/blob/eecbe6641f9ee10dc7d26b80d0398839fe6d3e00/test/tests/unit/backend.js#L5-L31
I didn't think it needed to be checked again in the integration test

Reply:
Ah got it. 👍🏻